Competition2017/src/competition/subsystems/autonomous/DriveToHopperCommandGroup.java

```diff
@@ -25,26 +34,41 @@ public DriveToHopperCommandGroup(XPropertyManager propManager,
         distanceFromTurningPointToHopper = propManager
                 .createPersistentProperty("Distance From Turning Point On Baseline To Hopper In Inches", 43.25);
         headingToFaceHopper = propManager.createPersistentProperty("Heading To Face Hopper In Degrees", 90);
+        redAllianceHeadingToFaceHopper = propManager.createPersistentProperty("Red alliance heading to face hopper", 0);
+        blueAllianceHeadingToFaceHopper = propManager.createPersistentProperty("Blue alliance heading to face hopper", 180);
```

Comment:
These are pretty standard values: the two horizontal headings. I'm thinking we can use constants instead of properties to ensure these are kept in sync... although the question of which angle is for which side.